docs/tutorial/part-one/index.md

```diff
@@ -19,7 +19,7 @@ When creating a new Gatsby site, you can use the following command structure to
 gatsby new [SITE_DIRECTORY_NAME] [URL_OF_STARTER_GITHUB_REPO]
 ```
 
-> 💡 See a list of the existing [**official and community starters**](/docs/gatsby-starters/)!
+> 💡 See a list of the existing [**official and community starters**](/docs/starters/)!
```

Comment:
We want the link to be resolve to `https://www.gatsbyjs.org/starters/` so it's `/starters/` 😄 	
In this case it's relative to the root of the page. With your link it then would be `https://www.gatsbyjs.org/docs/starters/`

```suggestion
> 💡 See a list of the existing [**official and community starters**](/starters/)!
```

Reply:
Woops. Makes sense. Sorry, I wasn't for sure where it was supposed to resolve to for sure lol.